Automattic/wp-calypso
# Settings: Remove horizontal navigation bar

Fixes https://github.com/Automattic/wp-calypso/issues/53774.

#### Changes proposed in this Pull Request

This PR removes the horizontal navigation bar from all the settings pages, since all pages are now accessible directly from the sidebar menu as of https://github.com/Automattic/jetpack/pull/20100.

Before | After
--- | ---
<img width="770" alt="Screen Shot 2021-06-18 at 18 17 03" src="https://user-images.githubusercontent.com/1233880/122590274-9c712d00-d061-11eb-9a1b-c787530f7376.png"> | <img width="758" alt="Screen Shot 2021-06-18 at 18 26 59" src="https://user-images.githubusercontent.com/1233880/122591113-cd059680-d062-11eb-96be-760121ea7ae4.png">

#### Testing instructions

- Go to any Settings page.
- Make sure the horizontal navigation bar is not rendered.
- Check further testings instructions from https://github.com/Automattic/jetpack/pull/20100 and make sure that all the settings pages are available in the Settings menu.
